crates/iceberg/src/scan.rs

```diff
@@ -209,8 +209,8 @@ pub struct FileScanTask {
 pub type ArrowRecordBatchStream = BoxStream<'static, crate::Result<RecordBatch>>;
 
 impl FileScanTask {
-    pub fn data_file(&self) -> ManifestEntryRef {
-        self.data_file.clone()
+    pub fn data_manifest_entry(&self) -> ManifestEntryRef {
```

Comment:
```suggestion
    pub fn data(&self) -> ManifestEntryRef {
```
How about just naming it as `data`? cc @marvinlanhenke What do you think?

Reply:
yes - i think this will work and still convey the meaning.